bellows/zigbee/zcl/foundation.py

```diff
@@ -101,7 +101,6 @@ class Discrete:
 DATA_TYPE_IDX = {
     t: tidx
     for tidx, (tname, t, ad) in DATA_TYPES.items()
-    if ad is Analog
```

Comment:
I'm OK with 2 or 3 as part of this PR. 2 would be deterministic. I'm worried that different implementations of dict in different versions of Python would cause 3 to work differently, unless you sorted by key or something in the comprehension. 2 would be more verbose, though. Up to you.

If you decide to do 2 or 3, please create an issue to remind me that 1 needs to be done at some point :)